app/styles/map-view.scss

```diff
@@ -8,6 +8,14 @@
   .is-list-view & {
     margin-left: -50%;
   }
+  .is-mobile.is-map-view & {
```

Comment:
Tænker du at lave desktop-håndteringen i et seperat issue? For det tror jeg ikke vi har lige nu.

Reply:
Jeg har oprettet en for desktop håndtering. :)